.rubocop.yml

```diff
@@ -15,7 +15,7 @@ AllCops:
     - 'lib/generators/**/templates/*.rb'
   DisplayCopNames: true
   SuggestExtensions: false
-  TargetRubyVersion: 2.6
+  TargetRubyVersion: 2.7
```

Comment:
While we are here, we can drop the line as per other PR I sent as it's inferred.

Reply:
Also, changelog says you dropped 2.7 support ;)

Reply:
> we can drop the line as per other PR I sent as it's inferred

I tried but for some reason it didn't work out 🤷‍♂️ I'll check again

Reply:
Okay, found the cause: I had Standard pinned to `~> 1.0.0`, which blocked me from using latest RuboCop. Now works.

Thanks!